app/konnect/gateway-manager/control-plane-groups/index.md

```diff
@@ -58,7 +58,10 @@ flowchart LR
 
 In a control plane group setup, each team still administers their own control plane, but the data plane nodes are shared. 
 
-The following diagram illustrates using a control plane group for a federated platform administrator model. In this example, Team Blue configures Control Plane Blue, which is then combined with the configuration from Team Green. In addition, the control plane group contains Control Plane Green, which is managed by a central platform team. The central team provides global plugin configuration, which is added to any configuration that teams Blue and Green provide.
+The following diagram illustrates using a control plane group for a federated platform administrator model. In this example:
+* Team Blue configures Control Plane Blue, which is then combined with the configuration from Team Green.
+* The control plane group also contains Control Plane Purple, which is managed by a central platform team.
+* The central team manages global plugin configuration in Control Plane Purple, which is added to any configuration that teams Blue and Green provide.
```

Comment:
```suggestion
* The central platform team manages global plugin configuration in Control Plane Purple, which is added to any configuration that teams Blue and Green provide.
```
Just because when I read this and then got to the purple part, I thought it meant to say Green because the first bullet was about Blue, second about Purple, so I thought the third would be about Green.